publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategyTest.java

```diff
@@ -87,6 +92,38 @@ void shouldDenyCuratorOnNonDegree(PublicationOperation operation)
                                    .create(publication, userInstance, uriRetriever)
                                    .allowsAction(operation));
     }
+
+    @Test
+    void isCuratorOnPublicationShouldReturnTrueWhenCuratorIsAssociatedWithPublication()
+        throws JsonProcessingException, UnauthorizedException {
+        var username = randomString();
+        var institution = randomUri();
+        var cristinId = randomUri();
+        var requestInfo = createUserRequestInfo(username, institution, getAccessRightsForCurator(), randomUri(), cristinId);
+        var publication = createNonDegreePublication(username, institution, cristinId);
+        var permissionStrategy = PublicationPermissionStrategy.create(publication,
+                                                                      RequestUtil.createUserInstanceFromRequest(requestInfo, identityServiceClient),
+                                                                      uriRetriever);
+        assertThat(permissionStrategy.isCuratorOnPublication(), is(equalTo(true)));
+    }
+
+    @Test
+    void shouldNotGivePermissionToUnpublishPublicationWithPublishedFilesWhenUserIsContributor()
```

Comment:
Does this test belong here? Shouldn't it be in `ContributorPermissionStrategy.java`?

Reply:
good point, will move